content/docs/console/continuous-delivery/jenkins.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Jenkins
+
+menu:
+    console:
+        parent: cont_delivery
+---
+
+This document will help you setup a [Jenkins Pipeline](https://jenkins.io/doc/book/pipeline/) to deploy a sample app to Azure using Pulumi.
+The source code will be hosted on GitHub just for the purpose of showing the GitHub integration between Jenkins and GitHub. Your source repository could be on any other VCS supported by Jenkins.
+
+Pulumi doesn't require any particular arrangement of stacks or workflow to work in a
+continuous integration / continuous deployment system. So the steps described here can be
+altered to fit into any existing type of deployment setup.
+
+## Prerequisites
+
+- A working installation of a recent version of Jenkins.
+- An account on the [Pulumi Console](https://app.pulumi.com).
+- The latest CLI. Installation instructions are [here]({{< relref "/docs/reference/install.md" >}}).
+- Setup a new project and stack using one of our [Get Started](https://www.pulumi.com/docs/quickstart/) guides or simply by running `pulumi new`
+and choosing one of the many templates that are available.
+- A bare repo and set the remote URL to be your GitHub project.
+
+## Sample Project
+
+An example project is located [here](https://github.com/pulumi/examples/azure-ts-appservice-springboot). You may download the project and upload it to your own repo to avoid having to clone the entire Pulumi Examples repo into your Jenkins workspace.
+
+## Stack and Branch Mappings
+The scripts below act on a hypothetical stack: `homer/acme/product-catalog-service-stack`.
+You can create a new stack by running `pulumi stack init` if you have already created a project.
+The source code for the stack is in a repository in GitHub and uses `TypeScript` as the language.
+
+**Note**: The names used above are purely for demonstration purposes only.
+You may choose a naming convention that best suits your organization.
+
+Alternatively, you can also run `pulumi new [template]` to create a template project.
+Learn more [here]({{< ref "/docs/reference/cli/pulumi_new.md" >}}).
+
+## `PULUMI_ACCESS_TOKEN`
+
+To login non-interactively in to the CLI, you will need to set the env var `PULUMI_ACCESS_TOKEN` as a build parameter when setting up the Jenkins build. To create a new access token, go the [Access Tokens](https://app.pulumi.com/account/tokens) page on the Pulumi Console.
+
+## Creating a New Jenkins Build
+
+### Classic UI
+
+The classic UI lets you create manual (UI) builds, as well as a Declarative pipeline-based build. There are several options available to cater to your specific setup, but for the purposes of this walkthrough, we will assume a single branch, declarative pipeline.
```

Comment:
Nit

```suggestion
The classic UI lets you create manual (UI) builds, as well as a declarative pipeline-based build. There are several options available to cater to your specific setup, but for the purposes of this walkthrough, we will assume a single branch, declarative pipeline.
```

Reply:
Actually, "Declarative" is a type of Jenkinsfile. The other is Scripted. That's what I wanted to highlighted here.